### DISCUSSION TITLE
Can't Open ZIP file for Windows.

### DISCUSSION THREAD

**Discussion by papaskwidd** [jerryuhoo/Fire#51] [Q&A]
Hello all. I have downloaded the win.zip file and it will not open - there is no installer. What am I missing? I could really use this plugin. I'm running Windows 11 and Studio One 6 Pro.
thanks for your time and attention
david

**Comment by jerryuhoo** [comment]
Unzip your downloaded file and move fire.VST3 to: ```C:\Program Files\Common Files\VST3```

**Reply by GentleDee** [reply]
FL Studio still doesn't find the file if I do what you just described. 
Any idea?
